setup.py

```diff
@@ -2,7 +2,8 @@
 from setuptools import setup
 
 setup(name='pyforce',
-    version='1.7.3', # be sure to update the version in pyforce.py too
```

Comment:
Didn't see any version number in pyforce.py, let me know if I missed something @alanjcastonguay

Reply:
That's no longer true. And you removed the comment as part of your change. Awesome~